src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp

```diff
@@ -359,9 +359,9 @@ static void test5_searchOutstandingMessagesTest()
 
     expectedStream << outstandingGUIDS.size() << " message GUID(s) found."
                    << bsl::endl;
-    float messageCount     = k_NUM_RECORDS / 3.0;
-    float outstandingRatio = float(outstandingGUIDS.size()) / messageCount *
-                             100.0;
+    const size_t messageCount     = k_NUM_RECORDS / 3;
+    const float  outstandingRatio = float(outstandingGUIDS.size()) /
+                                   float(messageCount) * 100.0f;
```

Comment:
Might be good to use C++ `static_cast`s here and below, not the C-style casts. We can do it in a separate PR if we don't want to increase the scope of this one